numpy/core/fromnumeric.py

```diff
@@ -804,7 +804,7 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     index_array : ndarray, int
         Array of indices that partition `a` along the specified axis.
         If `a` is one-dimensional, ``a[index_array]`` yields a partitioned `a`.
-        More generally, ``np.take_along_axis(a, index_array, axis=a)`` always
+        More generally, ``np.take_along_axis(a, index_array, axis=axis)`` always
```

Comment:
For educational purpuses, could we use a different name instead of axis for the variable? This could be confusing for those not so familiar with keyword arguments.

Reply:
@melissawm, I think that's a great idea. My background is math, not computer programming, and I remember being very confused when I would read documentation or code on the web because of how the same name would be used multiple times in different ways.  I actually wish that there was some sort of syntax highlighting or other convention for documentation to distinguish between the parts of the code that MUST be written as is, versus the parts of the code that are arbitrary names.  When I write example code for others, I typically do something like ```axis = YOUR_AXIS```, prepending YOUR to each arbitrary name to emphasize that it is whatever name they chose, but I suspect that someone who has taught computer programming knows a better technique.